### PR TITLE
[OSF] Allows opening the work template modal from a slash command

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Adds a `/templates` command to open the modal.

#### Ticket Link
N/A: OSF 

#### Related Pull Requests
- Has server changes https://github.com/mattermost/mattermost-server/pull/22246

#### Screenshots
image

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: actions/command.ts [hunk 1]
```diff
@@ -140,6 +141,15 @@ export function executeCommand(message: string, args: CommandArgs): ActionFunc {
 
             dispatch(openModal({modalId: ModalIdentifiers.PLUGIN_MARKETPLACE, dialogType: MarketplaceModal}));
             return {data: true};
+        case '/worktemplate': {
```
Comment (marianunez) [inline actions/command.ts:144]: Is `Work Template` the final name? It seems we were converging with `Templates`, wondering if this should be then just `/templates`?

Comment (JulienTant) [reply actions/command.ts:144]: I did wonder, but `/template(s)` is so generic 😅

Comment (julmondragon) [reply actions/command.ts:144]: I'd prefer `/templates` as well, but maybe more a call for UX / PO.

**Review by julmondragon — APPROVED** [review]
Thanks @JulienTant

**Review by marianunez — APPROVED** [review]

**Pull request merged**